Podcast/ActionSheetViewController.swift

```diff
@@ -219,12 +233,36 @@ class ActionSheetViewController: UIViewController, UITableViewDataSource, UITabl
         let option = options[indexPath.row]
         switch(option.type) {
         case .playerSettingsTrimSilence:
-            delegate?.didPressSegmentedControlForTrimSilence(selected: isSelected)
+            playerControlsDelegate?.didPressSegmentedControlForTrimSilence(selected: isSelected)
         case .playerSettingsCustomizePlayerSettings:
-            delegate?.didPressSegmentedControlForSavePreferences(selected: isSelected)
+            playerControlsDelegate?.didPressSegmentedControlForSavePreferences(selected: isSelected)
         default:
             break
         }
     }
 
+    func didPressSaveBlurb(for cell: ActionSheetCreateRecastBlurbTableViewCell, with blurb: String) {
+        guard let indexPath = optionTableView.indexPath(for: cell) else { return }
+        let option = options[indexPath.row]
+        switch(option.type) {
```

Comment:
remove parentheses - or since there's only one option you're checking you could just do `if case `